tools/cell_census_builder/validate.py

```diff
@@ -74,58 +78,85 @@ def validate_all_soma_objects_exist(soma_path: str, experiment_builders: List[Ex
         assert datetime.fromisoformat(census.metadata["created_on"])
         assert "git_commit_sha" in census.metadata
 
-        for name in [CENSUS_INFO_NAME, CENSUS_DATA_NAME]:
-            assert soma.Collection.exists(census[name].uri)
-            assert census[name].soma_type == "SOMACollection"
+        with tiledb.Group(census.uri) as census_group:
+            for name in [CENSUS_INFO_NAME, CENSUS_DATA_NAME]:
+                assert soma.Collection.exists(census[name].uri)
+                assert census[name].soma_type == "SOMACollection"
```

Comment:
this check is redundant and can be removed.  The `soma.Collection.exists()` peforms the same test....